content/en/docs/languages/java/sdk.md

```diff
@@ -72,8 +77,8 @@ component section includes:
 
 [OpenTelemetrySdk](https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-sdk/latest/io/opentelemetry/sdk/OpenTelemetrySdk.html)
 is the SDK implementation of
-[OpenTelemetry](https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/OpenTelemetry.html).
-It is a holder for top-level SDK components which makes it convenient to pass
+[OpenTelemetry](/docs/languages/java/api-components/#opentelemetry). It is a
```

Comment:
Change all of these to section-relative paths like this:

```suggestion
[OpenTelemetry](../api-components/#opentelemetry). It is a
```

Reply:
Noticed a bunch of other offending links in the java docs so I opened #5292 to address.